modules/genomic_browser/jsx/tabs_content/cnv.js

```diff
@@ -0,0 +1,160 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import FilterableDataTable from 'jsx/FilterableDataTable';
+import Loader from 'jsx/Loader';
+
+/**
+ * CNV Component.
+ *
+ * @description Genomic Browser CNV tab.
+ *
+ * @author Alizée Wickenheiser
+ * @version 1.0.0
+ *
+ */
+class CNV extends Component {
+  /**
+   * Constructor of component
+   * @param {object} props - the component properties.
+   */
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: {},
+      fieldOptions: {},
+      error: false,
+      isLoaded: false,
+    };
+    this.fetchData = this.fetchData.bind(this);
+    this.formatColumn = this.formatColumn.bind(this);
+  }
+
+  componentDidMount() {
+    this.fetchData()
+      .then(() => this.setState({isLoaded: true}));
+  }
+
+  /**
+   * Retrieve data from the provided URL and save it in state.
+   *
+   * @return {object}
+   */
+  fetchData() {
+    return fetch(window.location.origin + '/genomic_browser/CvnBrowser',
+      {credentials: 'same-origin'}
+    )
+      .then((resp) => resp.json())
+      .then((json) => {
+        // console.log(json);
+        const data = {
+          fieldOptions: json.fieldOptions,
+          Data: json.data.map((e) => Object.values(e)),
+          subprojects: json.subprojects,
+        };
+        this.setState({data});
+      }).catch((error) => {
+        this.setState({error: true});
+        console.error(error);
+      });
+  }
+
+  /**
+   * Modify behaviour of specified column cells in the Data Table component
+   *
+   * @param {string} column - column name
+   * @param {string} cell - cell content
+   * @param {array} rowData - array of cell contents for a specific row
+   * @param {array} rowHeaders - array of table headers (column names)
+   *
+   * @return {*} a formatted table cell for a given column
+   */
+  formatColumn(column, cell, rowData, rowHeaders) {
+    let reactElement = null;
+    if (true) {
+      switch (column) {
+        case 'PSCID':
+          const url = window.location.origin + '/' + rowData.DCCID + '/';
+          reactElement = (
```

Comment:
can be written on one line without parentheses

Reply:
@HenriRabalais I think one line looks ugly. It's similar in my mind as doing:
`if (true) myFunction();`
instead of:
```
if (true) {
   myFunction();
}
```
In any case I'll add but I find it really ugly.